src/lib/core/song-info.c

```diff
@@ -331,6 +331,19 @@ bt_song_info_tick_to_m_s_ms (const BtSongInfo * const self, const gulong tick,
   bt_song_info_time_to_m_s_ms (self, ts, m, s, ms);
 }
 
+gchar *
```

Comment:
Please add a docstring and add the function name to the docs (https://github.com/Buzztrax/buzztrax/blob/master/docs/reference/bt-core/buzztrax-core-sections.txt#L440).
Alternative would be to keep the function private and expose this through a new property on the class.

Reply:
Thanks for picking up on the missing docstring. I've added one and updated the sections file.

It is useful to know that I can use a readonly property to calculate these sort of derived values, I'll keep that in mind in future.

What's the purpose of buzztrax-core-sections.txt?

Reply:
*-sections.txt maps symbols that should be exposed to the docs to the right section. This can be also auto-generated, but then some cornercase cannot be addressed.